### DISCUSSION TITLE
Namron 4512737 Sending settings through Zigbee

### DISCUSSION THREAD

**Discussion by Hoppsan84** [Koenkk/zigbee2mqtt#13000] [Devices]
My Namron thermostat can't operate due to overcurrent when my floor heating starts. 
I want to try and disable the ACAlarmsMask to deactivate the function completely. 
According to the [manual](https://www.elektroimportoren.no/docs/lib/4512737-Brukerveiledning-5.pdf) this can be done sending a command through the Zigbee network, Problem is I can't figure out how to.

From the Manual:
ACAlarmsMask
0x0800 int16U,
Specifies which configurable alarms may be generated, only
Bit1: Current Overload is set, if set=0, then ACCurrentOverload=0, over
current will not be detected

To sum up, I want to se/change the ACAlarmsMask setting.

Br
Richard

**Comment by Hoppsan84** [comment]
Found the solution for this. 
This can be done in the Dev console.
When I figured out the manual it was really easy. 

Just put in:
Endpoint:1
Cluster:haElectricalMeasurment (stated in the manual)
Attribute: ACAlarmMask
Read.

Was set to 2 as standard. Is now set to 0. ACCurrentOverload is also set to 0.

Works like a charm.
